editor/src/clj/editor/collision_object.clj

```diff
@@ -409,12 +411,61 @@
                   (outline/gen-node-outline-keys (map (comp shape-type-label :shape-type)
                                                       shapes)))))))
 
+(defn convex-hull-scene
+  [_node-id convex-shape-data color]
+  (when (and (= (:shape-type convex-shape-data) :type-hull)
+             (not-empty (:data convex-shape-data)))
+    (let [points (->> convex-shape-data :data (partition 3) vec)
+          [min-coords max-coords] (reduce (fn [[min-point max-point] point]
+                                            [(mapv min min-point point) (mapv max max-point point)])
+                                          [(repeat Double/MAX_VALUE) (repeat Double/MIN_VALUE)]
+                                          points)
+          aabb (geom/coords->aabb max-coords min-coords)
+          center (-> aabb geom/aabb-center types/Point3d->Vec3)
+          point-count (count points)
+          pos-vtx-put! (fn [vb [x y z]] (scene-shapes/pos-vtx-put! vb x y z 0.0))
+          triangles (->> points
+                         (reduce-kv (fn [vb i point]
+                                      (let [next-point (nth points (inc i) (first points))]
+                                        (reduce pos-vtx-put! vb [center point next-point])))
+                                    (scene-shapes/->pos-vtx (* 3 point-count) :static))
+                         vtx/flip!)
```

Comment:
Given the fact that we expect a convex shape, we can use the center to triangulate the shape. I feel we should have a helper function for this, but I couldn't find it. Maybe we need to move this to `editor.scene-shapes`?